packages/shared/src/components/EditAltTextFlyout.tsx

```diff
@@ -0,0 +1,96 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MutableRefObject } from 'react';
+import {
+    ButtonEmphasis,
+    ButtonStyle,
+    Flyout,
+    FlyoutFooter,
+    FlyoutPlacement,
+    FormControl,
+    HelperPosition,
+    IconCheckMark16,
+    TextInput,
+} from '@frontify/fondue';
+import React from 'react';
+
+type EditAltTextFlyoutProps = {
```

Comment:
Also move the types into a `types.ts` file within `EditAltTextFlyout` folder

Reply:
👍